tools/autograd/templates/annotated_fn_args.py

```diff
@@ -0,0 +1,11 @@
+"""
+This file is needed for generating procedural tests required for
+testing __torch_function__. See tests/test_overrides.py.
+"""
+
+# flake8: noqa
+import torch
+
+annotated_args = {
+${annotated_args}
```

Comment:
In an earlier review comment, I asked to see some sample of what the generated file (`annotated_args`) looked like. Part of the reason for this was so I could get a sense for what was being exported. But another reason is to make it easier for people who want to look at `annotated_fn_args.py` to know what exactly they can get from this file. What entries are in the dictionary, what do they do? It also helps me, the code reviewer, understand if we are misusing the semantics of these entires :)

Reply:
I apologize, I must have forgotten.

Here is the entirety of the generated file: https://gist.github.com/hameerabbasi/e2ff4c39e1771458ec8a76e4d0e2bcfc

And here is a sample:

```python
"""
This file is needed for generating procedural tests required for
testing __torch_function__. See tests/test_overrides.py.
"""

# flake8: noqa
import torch

annotated_args = {
    ...
    torch._C._VariableFunctions.is_complex: [{'name': 'self', 'simple_type': 'Tensor'}],
    torch._C._VariableFunctions.is_nonzero: [{'name': 'self', 'simple_type': 'Tensor'}],
    torch._C._VariableFunctions.is_same_size: [{'name': 'self', 'simple_type': 'Tensor'}, {'name': 'other', 'simple_type': 'Tensor'}],
    torch._C._VariableFunctions.is_signed: [{'name': 'self', 'simple_type': 'Tensor'}],
    torch._C._VariableFunctions.kl_div: [{'name': 'self', 'simple_type': 'Tensor'}, {'name': 'target', 'simple_type': 'Tensor'}],
    ...
}
```